src/nnsight/tracing/Bridge.py

```diff
@@ -23,6 +25,7 @@ def __init__(self) -> None:
         self.id_to_graph: Dict[int, "Graph"] = OrderedDict()
         # Stack to keep track of most inner current graph
         self.graph_stack: List["Graph"] = list()
+        self.bridged_nodes: defaultdict[Node, List[Node]] = defaultdict(lambda: list())
```

Comment:
Can you confirm there isnt any issue with this holding onto Nodes and therefore memory longer than needed?

Reply:
Are you saying we could reinitialize this storing variable every time a new intervention graph is being entered?